### PR TITLE
new ioctl command for update

### PR DESCRIPTION
**Pull request by RuneAkko**
#1776

### CODE REVIEW

File: ioctl/newcmd/update/update.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+				cmdString = "curl --silent https://raw.githubusercontent.com/iotexproject/" + "iotex-core/master/install-cli.sh | sh -s \"unstable\""
+
+			}
+
+			output.PrintResult(fmt.Sprintf("Downloading the latest %s version ...\n", versionType))
+			err := exec.Command("bash", "-c", cmdString).Run()
+			if err != nil {
+				return output.NewError(output.UpdateError, "failed to update ioctl", nil)
+			}
+			output.PrintResult("ioctl is up-to-date now.")
```
Comment (CoderZhi) [inline ioctl/newcmd/update/update.go]: TODO: add translations

File: ioctl/newcmd/update/update.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+			default:
+				output.NewError(output.FlagError, "invalid version-type flag:"+versionType, nil)
+			case "stable":
+				cmdString = "curl --silent https://raw.githubusercontent.com/iotexproject/" + "iotex-core/master/install-cli.sh | sh"
+			case "unstable":
+				cmdString = "curl --silent https://raw.githubusercontent.com/iotexproject/" + "iotex-core/master/install-cli.sh | sh -s \"unstable\""
+
+			}
+
+			output.PrintResult(fmt.Sprintf("Downloading the latest %s version ...\n", versionType))
```
Comment (CoderZhi) [inline ioctl/newcmd/update/update.go]: TODO: add translations

File: ioctl/newcmd/update/update.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	uses = map[config.Language]string{
+		config.English: "update [ALIAS|ADDRESS]",
+		config.Chinese: "update [别名|地址]",
+	}
+)
+
+func NewUpdateCmd(c ioctl.Client) *cobra.Command {
+	var versionType string
+	use, _ := c.SelectTranslation(uses)
+	short, _ := c.SelectTranslation(shorts)
```
Comment (CoderZhi) [inline ioctl/newcmd/update/update.go]: TODO: create function `func MustSelectTranslation(map[config.Language]string) string`

Comment (RuneAkko) [reply ioctl/newcmd/update/update.go]: > TODO: create function `func MustSelectTranslation(map[config.Language]string) string`

Is it means that I need to create a MustFunction which can recover the panic "failed to pick a translation" to replace `MustSelectTranslation` ?

Comment (CoderZhi) [reply ioctl/newcmd/update/update.go]: no, you can just leave a todo here.

**Pull request merged**